tests/integration/components/public/ticket-list-test.js

```diff
@@ -1,14 +1,50 @@
 import { test } from 'ember-qunit';
+import Ember from 'ember';
 import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
 import hbs from 'htmlbars-inline-precompile';
 
-moduleForComponent('public/ticket-list', 'Integration | Component | public/ticket list');
+const { Object, ArrayProxy } = Ember;
```

Comment:
@niranjan94 A.create fails 3  the tests, the component and the moment and a global failure.
Am I missing something? I just replaced ArrayProxy with A.